examples/nlp/text_classification_tf_keras/model_def.py

```diff
@@ -0,0 +1,35 @@
+import tensorflow as tf
+from tensorflow import keras
+from tensorflow.keras import layers
+from determined.keras import TFKerasTrial, TFKerasTrialContext, InputData
+import data
+
+
+class MultiTextClassificationTrial(TFKerasTrial):
+    def __init__(self, context: TFKerasTrialContext) -> None:
+        self.context = context
+
+    def build_model(self):
+        model = tf.keras.Sequential([
+            data.create_vectorization_layer(),
+            layers.Embedding(10000, self.context.get_hparam("embedding_dim")),
+            layers.Dropout(0.2),
+            layers.GlobalAveragePooling1D(),
+            layers.Dropout(0.2),
+            layers.Dense(self.context.get_hparam("dense1"))])
```

Comment:
Style-wise, can we put this set of parentheses on an extra line?

Reply:
done